### ISSUE TITLE
Windows build FTBFS: ... ghc-9.0.2\\": does not exist

### ISSUE THREAD

**Issue by yarikoptic** [datalad/git-annex#130]
```shell
Extracted 11790 files
Extracted total of 11790 files from ghc-9.0.2.tar
C:\Users\runneradmin\AppData\Local\Programs\stack\x86_64-windows\ghc-9.0.2-tmp-2a6be56bfff1c7a0\ghc-9.0.2\: renameDirectory:pathIsDirectory:CreateFile "\\\\?\\C:\\Users\\runneradmin\\AppData\\Local\\Programs\\stack\\x86_64-windows\\ghc-9.0.2-tmp-2a6be56bfff1c7a0\\ghc-9.0.2\\": does not exist (The system cannot find the file specified.)
Error: Process completed with exit code 1.
```

**Comment by jwodder** [comment]
@yarikoptic Running the Windows build right now succeeds where the previous builds failed (and then they go on to fail during compilation proper due to something wrong with a preprocessor directive in the code).  I suspect the failures were fixed by dddaec1b740197b8654ae4242774dab2b888c6f9.

**Comment by yarikoptic** [comment]
hm, that commit is from "2 days ago" whenever the failing like that build https://github.com/datalad/git-annex/runs/7644770316?check_suite_focus=true from "13 hours ago"... a little odd, but may be that commit was not pushed for a while...  indeed -- there was a merge after the commit we built d297e9ba7 which brought that fix in. Thanks @jwodder !
